### PR TITLE
Section 3: Movement and Collisions

### PR DESCRIPTION
**Pull request by thompsop1sou**
This PR updates the movement and collisions section of the tutorial to match the previous sections.

### CODE REVIEW

**Comment by thompsop1sou** [comment]
@defreez I think this PR is ready to be reviewed. I walked through section 3 and it seemed to work with no issues. (The first time I ran it, there was a warning about a wrong ID in the map.tscn scene, but I think it fixed itself.)

**Comment by defreez** [comment]
There are conflicts with 01 and 02 on this branch that we will need to resolve before we can merge.

**Comment by thompsop1sou** [comment]
Conflicts should be resolved now.

File: docs/first_semantic_game/03.movement_and_collisions.rst [hunk 1]
```diff
@@ -1,10 +1,9 @@
 Movement and Collisions
 =======================
 
-In the last section, we saw how to create a simple scene that allowed us to connect Godot up to
-Sematic-Kernel and converse with a GPT-4 powered NPC. Now, our goal is to take this one step
-further by embedding that NPC in a slightly more developed game world, one with movement and
-collisions.
+In the last section, we saw how to connect Godot up to Sematic-Kernel and create two GPT-4 powered
+NPCs that could talk with each other. Now, our goal is to take this idea further by embedding such
+an NPC in more developed game world, one with movement and collisions.
```
Comment (defreez) [inline docs/first_semantic_game/03.movement_and_collisions.rst:6]:
```suggestion
an NPC in a more developed game world, one with movement and collisions.
```

File: docs/first_semantic_game/03.movement_and_collisions.rst [hunk 2]
```diff
@@ -133,32 +141,37 @@ property:
...
+Seraphis shown below. Add it to the sprite by dragging the file onto the ``Texture`` property in
+the inspector. (Alternately, you can click on the ``Texture`` property and select "Load" from the
+drop-down menu. Then find and open *seraphis.png*.)
 
-Finally, add a ``Sprite2D`` node as a child of the ``Seraphis`` node. The picture of Seraphis is
-provided in the file *seraphis.png*. Add it to the sprite by dragging the file onto the ``Texture``
-property in the inspector. (Alternately, you can click on the ``Texture`` property and select "Load"
-from the drop-down menu. Then find and open *seraphis.png*.)
+.. image:: seraphis.png
+   :alt: The Gralk sprite
```
Comment (defreez) [inline docs/first_semantic_game/03.movement_and_collisions.rst:153]:
```suggestion
   :alt: The Seraphis sprite

```

**Review by defreez — DISMISSED** [review]
LGTM just a couple of tiny suggestions.

**Review by defreez — APPROVED** [review]

**Comment by defreez** [comment]
@thompsop1sou Feel free to merge when you are ready

**Pull request merged**